CHANGES.md

```diff
@@ -32,29 +32,50 @@ replace with next unreleased version
 	- [JIRA View of Open Issues](https://codice.atlassian.net/issues/?jql=project%3DCodice%20Alliance%20AND%20issuetype%20%3D%20bug%20%20AND%20status%20%3D%20Open%20AND%20affectedVersion%20%3D%20VERSION_NUMBER%20AND%20fixVersion%20!%3D%20VERSION_NUMBER%20ORDER%20BY%20priority) affecting this version.
  -->
 
-## 0.2.1
+## 0.3.0
 	Release Date: pending
 
 ### NOTES
 
-- Built on [DDF v.2.10.1](https://github.com/codice/ddf/blob/master/CHANGES.md#2101)
+- Built on [DDF v.2.11.1](https://github.com/codice/ddf/blob/master/CHANGES.md#2110)
```

Comment:
More likely 0.3.0 will be built on 2.11.0. It's all just guesswork at this point, though.